plugins/continuous-toolbox/src/ContinuousMetrics.ts

```diff
@@ -12,24 +12,26 @@ import * as Blockly from 'blockly/core';
 
 /** Computes metrics for a toolbox with an always open flyout. */
 export class ContinuousMetrics extends Blockly.MetricsManager {
-  /** @override */
-  constructor(workspace) {
-    super(workspace);
-  }
   /**
    * Computes the viewport size to not include the toolbox and the flyout.
    * The default viewport includes the flyout.
-   * @override
+   *
+   * @param getWorkspaceCoordinates True to get the view metrics in workspace
+   *     coordinates, false to get them in pixel coordinates.
+   * @returns The width, height, top and left of the viewport in either
+   *     workspace coordinates or pixel coordinates.
    */
-  getViewMetrics(getWorkspaceCoordinates = undefined) {
+  override getViewMetrics(
+    getWorkspaceCoordinates?: boolean,
```

Comment:
I think you could have changed this to 
```suggestion
    getWorkspaceCoordinates = false,
```
and effected the same type change while making the default clear.  Or do you want to be able to distinguish between "no value supplied" and "value supplied is `false`" cases?

Reply:
Done; I think this was probably just matching how it was done in the superclass, which should perhaps be similarly adjusted.